### PR TITLE
Add feedback overlay

### PR DESCRIPTION
**Pull request by TiagoGraf89**

### CODE REVIEW

**Comment by healthgatewaybc** [comment]
WebClient Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=webclient_pr_217)

File: Apps/WebClient/src/ClientApp/app/services/interfaces.ts [hunk 1]
```diff
@@ -7,6 +7,7 @@ import ImmsData from "@/models/immsData";
 import PatientData from "@/models/patientData";
 import RequestResult from "@/models/requestResult";
 import UserProfile from "@/models/userProfile";
+import UserFeedback from "../models/userFeedback";
```
Comment (FuriousLlama) [inline Apps/WebClient/src/ClientApp/app/services/interfaces.ts]: nit: use @ instead of ...

Comment (TiagoGraf89) [reply Apps/WebClient/src/ClientApp/app/services/interfaces.ts]: Fixed

**Comment by healthgatewaybc** [comment]
WebClient Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=webclient_pr_217)

**Pull request merged**